primitives/externalities/src/lib.rs

```diff
@@ -42,26 +40,40 @@ pub trait Externalities: ExtensionStore {
 	fn storage(&self, key: &[u8]) -> Option<Vec<u8>>;
 
 	/// Get storage value hash. This may be optimized for large values.
-	fn storage_hash(&self, key: &[u8]) -> Option<H256>;
+	fn storage_hash(&self, key: &[u8]) -> Option<Vec<u8>>;
```

Comment:
I suppose we don't make hasher as a associated type because we want to have this trait being able to be a trait object isn't it ?

Reply:
Yes